example_async_and_test/lib/main.dart

```diff
@@ -34,15 +34,22 @@ class MyApp extends StatelessWidget {
 // Note: It must extend from Model.
 abstract class AbstractModel extends Model {
   int get counter;
+  bool get disabled;
```

Comment:
Could we please use `enabled` instead of `disabled`? Positive statements are generally easier to read than negative statements.